.drone.star

```diff
@@ -313,3 +319,153 @@ def appBuilds(ctx):
             ],
         },
     }]
+
+def ocisService():
+    environment = {
+        "OCIS_URL": "https://ocis:9200",
+        "OCIS_INSECURE": "true",
+        "OCIS_LOG_LEVEL": "error",
+        "IDM_ADMIN_PASSWORD": "admin",
+        "PROXY_ENABLE_BASIC_AUTH": True,
+        "WEB_ASSET_APPS_PATH": "/var/lib/ocis/web/apps",
+        "WEB_UI_CONFIG_FILE": "/var/lib/ocis/web/config.json",
+    }
+
+    app_build_steps = [
+        {
+            "name": "build-web",
+            "image": OC_CI_NODEJS,
+            "commands": [
+                "pnpm build",
+                "mkdir -p /apps",
+                "mv packages/web-app-draw-io/dist /apps/draw-io",
+            ],
+            "volumes": [
+                {
+                    "name": "apps",
+                    "path": "/apps",
+                },
+            ],
+        },
+    ]
```

Comment:
we can have separate function for this step and generate build steps with `for app in APPS_COVERED_E2E:`

Reply:
I don;t want to put this to` for app in APPS_COVERED_E2E` 
It's a common step to build all the web extensions with a single command.